src/log/formats/csv.ts

```diff
@@ -9,7 +9,7 @@ export default function csv(log: LogEntry[]): string {
       entry.time.toISOString(),
       eventTypeString.get(entry.eventType),
       entry.ip,
-      entry.userAgent
+      `"${entry.userAgent}"`
```

Comment:
This might cause issues now if there's quotes in the string. I think? escaping `"` is done by doubling it? We could also switch to `csv-stringify` or something

Reply:
True. That gives a `CSV_INVALID_CLOSING_QUOTE` error. I'll see about using `csv-stringify` instead of doing this manually.